crates/sui-framework/tests/vec_map_tests.move

```diff
@@ -121,4 +121,17 @@ module sui::vec_map_tests {
         }
     }
 
+    #[test]
+    fun return_list_of_keys() {
+        let m = vec_map::empty();
+
+        assert!(vec_map::keys(&m) == vector::empty(), 0);
+
+        vec_map::insert(&mut m, 1, true);
+        vec_map::insert(&mut m, 5, false);
+
+        let expected_keys = vector::singleton(5);
+        vector::push_back(&mut expected_keys, 1);
+        assert!(vec_map::keys(&m) == expected_keys, 1);
```

Comment:
```suggestion
        assert!(vec_map::keys(&m) == vector[1, 5], 1);
```

Reply:
This will let you kill the `insert`'s above

Reply:
thanks!